jest.config.js

```diff
@@ -9,13 +9,15 @@ module.exports = {
         },
     },
     coveragePathIgnorePatterns: ["/node_modules/", "/server/"],
-    setupFiles: ["<rootDir>/test/jest.setup.ts"],
+    setupFilesAfterEnv: ["<rootDir>/test/jest.setup.ts"],
     coverageReporters: ["json", "lcov", "text", "text-summary"],
     moduleNameMapper: {
         "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$":
             "<rootDir>/test/mocks.ts",
         "\\.(css|less|scss|html)$": "<rootDir>/test/mocks.ts",
+        "^@(Test)(.*)$": "<rootDir>/test/$2",
         "^@([A-Z].*)$": "<rootDir>/src/$1",
     },
     preset: "ts-jest",
+    snapshotSerializers: ["enzyme-to-json/serializer"],
```

Comment:
enzyme-to-json/serializer bu paketi package.json'a eklememişiz. npm install yaparken mutlaka --save parametresini geçelim

Reply:
package.json'a eklenmiş ama dependencies altındaydı. devDependencies altına taşıdım.